### PR TITLE
Optimise

### PR DESCRIPTION
**Pull request by mcmenaminadrian**
Key thing here is get rid of the control buffer - it seems to work but please do check it over as you understand this code better than me

### CODE REVIEW

**Comment by mcmenaminadrian** [comment]
I've also added two new bread types to be true to the motherland

File: SplitFiles/bread.c [hunk 1]
```diff
@@ -55,9 +55,9 @@ BreadState* get_bread(BreadBin* bin, unsigned char typeLetter) {
     }
     BreadState* newBread = malloc(sizeof(BreadState));
     newBread->temperature = 0;
-    newBread->moisture = type->moisture + (rand() % type->moistureVariability);
+    newBread->moisture = type->moisture + (rand() & type->moistureVariability);
```
Comment (ConorMcPersonal) [inline SplitFiles/bread.c:58]: The big drag on bread creation is the sound - saving a couple of cycles here at the expense of imposing a hidden constraint on the code (variability must be a power of 2 - 1) doesn't feel like a great trade-off.  This late in the day it's fine I suppose, but this feels like change for its own sake.

Comment (mcmenaminadrian) [reply SplitFiles/bread.c:58]: Any sort of division eats cycles on any of these older machines and modular arithmetic is even worse, though I get your point

**Comment by ConorMcPersonal** [comment]
I've simplified the control code and removed the key click - that made the controls feel a bit spongy to me and seemed to interfere with slot sounds.

I approve your changes

**Comment by mcmenaminadrian** [comment]
I’ve just gone to bed! If you are happy with it please merge.

**Pull request merged**